experiments/functional/cvr-scaleup/test.yml

```diff
@@ -225,6 +238,26 @@
           delay: 10
           retries: 50
 
+        - block: 
+
+            - name: Getting the taget name 
+              shell:  kubectl get pod -n {{ operator_ns }} -l openebs.io/persistent-volume={{ pv.stdout }} -o jsonpath='{.items[0].metadata.name}' 
+              register: target_name
+
+            - name: Checking for reconstructing state of newely created cvr 
+              shell: kubectl get cvr {{ pv.stdout }}-{{ csp_name.stdout_lines[0] }} -n {{ operator_ns }} -o jsonpath='{.status.phase}'
+              register: new_cvr_state
+              until: "'ReconstructingNewReplica' in new_cvr_state.stdout"
+              delay: 10
+              retries: 100
+
+            - name: Restarting target
+              shell: kubectl delete pod {{ target_name.stdout }} -n {{ operator_ns }}
+              register: target_delete_status
+              failed_when: "'deleted' not in target_delete_status.stdout"
```

Comment:
@shashank855  instead of failed_when use the until condition to check the older target is removed.